docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -6,8 +6,8 @@ When loading studies into cBioPortal it is possible for warnings to occur that a
 The cBioPortal scripts package provides a method to update the `gene` and `gene_alias` tables. This requires the latest version of the NCBI Gene Info.
```

Comment:
@yichaoS I don't think we need the "This requires the latest version of the NCBI Gene Info." sentence anymore.

Reply:
Thx. Removed.